SIL.Core/SIL.Core.csproj

```diff
@@ -1,585 +1,29 @@
-    <Compile Include="ObjectModel\WrapperEqualityComparer.cs" />
-    <Compile Include="PlatformUtilities\NativeException.cs" />
-    <Compile Include="Progress\Commands\AsyncCommand.cs" />
-    <Compile Include="Progress\BackgroundWorkerState.cs" />
-    <Compile Include="Progress\Commands\BasicCommand.cs" />
-    <Compile Include="Progress\ConsoleProgressState.cs" />
-    <Compile Include="Progress\IProgressIndicator.cs" />
-    <Compile Include="Progress\Progress.cs" />
-    <Compile Include="Progress\NullProgressState.cs" />
-    <Compile Include="Progress\ProgressState.cs" />
-    <Compile Include="Progress\XslTransformWithProgress.cs" />
-    <Compile Include="Properties\Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Resources.resx</DependentUpon>
-    </Compile>
-    <Compile Include="Properties\Settings.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTimeSharedInput>True</DesignTimeSharedInput>
-      <DependentUpon>Settings.settings</DependentUpon>
-    </Compile>
-    <Compile Include="Reflection\ReflectionHelper.cs" />
-    <Compile Include="Reporting\AnalyticsEventSender.cs" />
-    <Compile Include="Reporting\ConfigurationException.cs" />
-    <Compile Include="Reporting\ConsoleErrorReporter.cs" />
-    <Compile Include="Reporting\ConsoleExceptionHandler.cs" />
-    <Compile Include="Reporting\ErrorReport.cs" />
-    <Compile Include="Reporting\ErrorReportSettings.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTimeSharedInput>True</DesignTimeSharedInput>
-      <DependentUpon>ErrorReportSettings.settings</DependentUpon>
-    </Compile>
-    <Compile Include="Reporting\ExceptionHandler.cs" />
-    <Compile Include="Reporting\ExceptionHelper.cs" />
-    <Compile Include="Reporting\Logger.cs" />
-    <Compile Include="Reporting\ReportingSettings.cs" />
-    <Compile Include="Reporting\UsageReporter.cs" />
-    <Compile Include="IO\SFMReader.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Code\RetryUtility.cs" />
-    <Compile Include="Annotations\Annotation.cs" />
-    <Compile Include="Annotations\IAnnotatable.cs" />
-    <Compile Include="Spelling\WordTokenizer.cs" />
-    <Compile Include="IO\TempFileForSafeWriting.cs" />
-    <Compile Include="Text\ApproximateMatcher.cs" />
-    <Compile Include="Text\LanguageForm.cs" />
-    <Compile Include="Text\MultiTextBase.cs" />
-    <Compile Include="Threading\GlobalMutex.cs" />
-    <Compile Include="UiBindings\IBindableControl.cs" />
-    <Compile Include="UiBindings\ICountGiver.cs" />
-    <Compile Include="UiBindings\IDisplayStringAdaptor.cs" />
-    <Compile Include="Unicode\CharacterUtils.cs" />
-    <Compile Include="IO\FileRelatedStrings.Designer.cs">
-      <DependentUpon>FileRelatedStrings.resx</DependentUpon>
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-    </Compile>
-    <Compile Include="Properties\AssemblyInfo.cs" />
```

Comment:
ok, so I asked @andrew-polk about this on his PR. How does this work now and why won't I run into problems if I'm changing branches which have different sets of files?

Reply:
So given the discussion on the other PR I'm happy about this change now.